File: 프로그래머스/1/12940. 최대공약수와 최소공배수/최대공약수와 최소공배수.cpp
#include <string>
#include <vector>
#include <cmath>

using namespace std;

vector<int> solution(int n, int m) {
    vector<int> answer;
    
    int gcd = 1;
    
    int maxNum = max(n, m);
    for(int i=1; i<=maxNum; i++){
        if(n%i == 0 && m%i == 0)
            gcd = i;
    }
    
    int cm = n*m/gcd;
    
    answer.push_back(gcd);
    answer.push_back(cm);
    
    return answer;
}

File: 프로그래머스/1/12940. 최대공약수와 최소공배수/최대공약수와 최소공배수.js
function solution(n, m) {
    var answer = [];
    
    let gcd = 1;
    
    let maxNum = Math.max(n, m);
    for(let i=1; i<=maxNum; i++){
        if(n%i === 0 && m%i === 0)
            gcd = i;
    }
    
    let cm = n*m/gcd;
    
    answer.push(gcd);
    answer.push(cm);
    
    return answer;
}